scripts/jenkins-release.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+[ -d venv ] || virtualenv venv
+pip install -e .
+
+if [ "$ENVIRONMENT" = "Select one" -o "$ENVIRONMENT" = "" ]; then
```

Comment:
Agreed in real life to rename this variable to STAGE and then look at renaming STAGE to ENVIRONMENT and ENVIRONMENT to ENVIRONMENT_CONTEXT.